uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -148,9 +148,9 @@ public ControllerHelixManager(SourceKafkaClusterValidationManager srcKafkaValida
     // 2. establishing connection with server;
     // 3. getting next data snippet from server.
     _requestConfig = RequestConfig.custom()
-        .setConnectionRequestTimeout(3000)
-        .setConnectTimeout(3000)
-        .setSocketTimeout(3000)
+        .setConnectionRequestTimeout(30000)
```

Comment:
why we increase the timeout here? could you add more comments here?

Reply:
I saw increased http get request time and increased the timeout. (now we know it's due to zookeeper instability)